_i18n/en/resources/user-guides/mine-to-pool.md

```diff
@@ -1,107 +1,194 @@
-{% assign version = '1.1.1' | split: '.' %}
+{% assign version = '1.2.0' | split: '.' %}
 {% include disclaimer.html translated="true" version=page.version %}
-## Selecting a pool
+## Wallet
 
-There are many pools to choose from, a list is available at
-[moneropools.com](https://moneropools.com). Mining on a larger pool could mean
-more frequent payouts, but mining on a smaller pool helps to keep the network
-decentralized.
+Before starting, you already need to have a wallet configured and
+working. The pool needs to know your wallet address to be able to send
+payments there.
 
-## Selecting a CPU miner
+<!-- TODO: Link to guide. -->
```

Comment:
Wasn't sure which guide to link to here, or how to do it. The [Accepting Monero](https://web.getmonero.org/get-started/accepting/) guide and [Getting started with the CLI wallet](https://web.getmonero.org/resources/user-guides/monero-wallet-cli.html) both seemed like good choices.

Reply:
I think linking to *Accepting Monero* would be fine. You can link to it using `{{ site.baseurl }}/get-started/accepting/`